src/Hopper/Internal/LoweredCore/ClosureConvertedANF.hs

```diff
@@ -269,11 +267,10 @@ data SymbolRegistryCC =
 
 lookupClosureCodeId :: SymbolRegistryCC -> ClosureCodeId-> Either String ClosureCodeRecordCC
 lookupClosureCodeId (SymbolRegistryCC _thk closMap _vals) codeid =
-      maybe  (Left $ "failed closure code lookup " ++ show codeid) (Right) $
+      maybe (Left $ "failed closure code lookup `" ++ show codeid ++ "`") Right $
         Map.lookup codeid closMap
+
 lookupThunkCodeId :: SymbolRegistryCC -> ThunkCodeId-> Either String ThunkCodeRecordCC
 lookupThunkCodeId (SymbolRegistryCC thkmap _closMap _vals) thud =
-      maybe (Left $ "failed thunk code lookup " ++ show thud ) (Right) $
+      maybe (Left $ "failed thunk code lookup `" ++ show thud ++ "`") Right $
```

Comment:
ok, this is the ` ` within the error / string convension bit,  goes back to the we need to investigate current pretty printing tooling out there and/or invest in building our own hybrid in that space